person.rb

```diff
@@ -1,8 +1,8 @@
 require './nameable'
-
+require './book'
 # Class Person
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals, :parent_permession
```

Comment:
- [ ] At that point, I do not think you are going to need `:parent_permession`. Kindly remove it if it's not necessary. 🚮🗑️